operations/finance.md

```diff
@@ -347,80 +404,52 @@ Returns all accounting items of the enterprise that were consumed \(posted\) or
-| `Amount` | [Amount value](finance.md#amount-value) | required | Item's amout, negative amount represents either rebate or a payment. |
-
-#### Accounting item type
+| `AccountingState` | string [Accounting state](#accounting-item-state) | required | Accounting state of the item. |
+| `Data` | object [Order item data](#order-item-data) | required | Additional data specific to particular order item. |
 
-* `ServiceRevenue`
-* `ProductRevenue`
-* `AdditionalRevenue`
-* `Payment`
+#### Payment item
 
-#### Accounting item subtype
-
-* Revenue subtypes
-  * `CancellationFee`
-  * `Rebate`
-  * `Deposit`
-  * `ExchangeRateDifference`
-  * `CustomItem`
-  * `Surcharge`
-  * `SurchargeDiscount`
-  * `SpaceOrder`
-  * `ProductOrder`
-  * `Other`
-* Payment subtypes
-  * `CreditCard`
-  * `Invoice`
-  * `Cash`
-  * `Unspecified`
-  * `BadDebts`
-  * `WireTransfer`
-  * `ExchangeRateDifference`
-  * `ExchangeRoundingDifference`
-  * `BankCharges`
-  * `Cheque`
-  * `Other`
-
-#### Accounting item substate
-
-* Payment substates
-  * `Pending`
-  * `Verifying`
-  * `Charged`
-  * `Canceled`
-  * `Failed`
+| Property | Type | Contract | Description |
+| --- | --- | --- | --- |
+| `Id` | string | required | Unique identifier of the item. |
+| `AccountId` | string | required | Unique identifier of the account (for example [Customer](customers.md#customer)) the item belongs to. |
+| `BillId` | string | optional | Unique identifier of the [Bill](#bill) the item is assigned to. |
+| `AccountingCategoryId` | string | optional | Unique identifier of the [Accounting category](#accounting-category) the item belongs to. |
+| `Amount` | [Amount value](#amount-value) | required | Item's amount, negative amount represents either rebate or a payment. |
+| `Notes` | string | optional | Additional notes. |
+| `ConsumedUtc` | string | required | Date and time of the item consumption in UTC timezone in ISO 8601 format. |
+| `ClosedUtc` | string | optional | Date and time of the item bill closure in UTC timezone in ISO 8601 format. |
+| `AccountingState` | string [Accounting state](#accounting-item-state) | required | Accounting state of the item. |
+| `State` | string [Payment item state](#payment-item-state) | required | Payment state of the item. |
+| `Data` | object [Payment item data](#payment-item-data) | required | Additional data specific to particular payment item. |
 
-#### Currency value
+#### Credit card transaction
 
 | Property | Type | Contract | Description |
 | --- | --- | --- | --- |
-| `Currency` | string | required | ISO-4217 code of the [Currency](configuration.md#currency). |
-| `Value` | number | optional | Amount in the currency. |
+| `PaymentId` | string | required | Unique identifier of the [Payment item](#payment-item). |
+| `ChargedAmount` | [Amount](#amount-value) | required | Charged amount of the transaction. |
+| `SettledAmount` | [Amount](#amount-value) | optional | Settled amount of the transaction. |
```

Comment:
Why not to order them the same they are in the DTO -> they are going to be in the json? I know you put required on top, but would be nice the rest to stay the same.